cmd/net_test.go

```diff
@@ -175,6 +175,21 @@ func TestGetAPIEndpoints(t *testing.T) {
 	}
 }
 
+// Ask the kernel for a free open port.
+func getFreePort() string {
```

Comment:
Rename this function to `mustGetFreePort()`.

Reply:
I would defer to make such changes in a dependency removal PR.